src/lib/ng-packagr.ts

```diff
@@ -15,13 +15,15 @@ import * as log from './util/log';
 import { NgPackageData } from './model/ng-package-data';
 
 
-/** CLI arguments passed to `ng-packagr` and `ngPackage()`. */
-export interface NgPackagrCliArguments {
+/**
+ * Arguments passed to `ng-packagr` binary (from CLI) and `createNgPackage()`.
+ */
+export interface CliArguments {
   /** Path to the 'ng-package.json' file */
```

Comment:
This comment should probably be updated since it could be a js file now as well.